test/lib/render-helpers.js

```diff
@@ -46,3 +49,18 @@ export function mountWithRouter (component, store = {}, pathname = '/') {
 
   return mount(<Wrapper />, createContext())
 }
+
+export function renderWithProvider (component) {
+
+  const { store } = component.props
```

Comment:
Interesting idea :thinking: Passing the store in as a prop.

Any reason to not pass it as a separate parameter to `renderWithProvider` instead?

Reply:
It was just to make it look cleaner and more React like, `renderWithProvider(<UnconnectedAccountAlert store={store} />)` vs `renderWithProvider(<UnconnectedAccountAlert />, store)`

Reply:
I can go either way, just though it was a good syntactical sugar.

Reply:
Ah I see. I can see why you'd prefer the first version. I prefer the second because it keeps the component under test separate from test environment setup, but I wasn't going to block on it